addons/html_builder/static/src/website_builder/website_mass_mailing/mailing_list_subscribe_option.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.MailingListSubscribeOption">
+
+    <BuilderRow label.translate="Newsletter">
+        <BuilderSelect dataAttributeAction="'listId'">
+            <t t-foreach="state.mailingLists" t-as="item" t-key="item.id">
+                <BuilderSelectItem dataAttributeActionValue="item.id.toString()">
+                    <t t-out = "item.name"/>
+                </BuilderSelectItem>
+            </t>
+            <t t-if="!state.mailingLists.length">
+                <BuilderSelectItem>None</BuilderSelectItem>
+            </t>
+        </BuilderSelect>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Display Thanks Message">
+        <BuilderCheckbox dataAttributeAction="'toggleThanksMessage'" dataAttributeActionValue="'true'" action="'toggleThanksMessage'"/>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Placeholder">
+        <BuilderTextInput attributeAction="'placeholder'" applyTo="'.s_newsletter_subscribe_form_input'"/>
+    </BuilderRow>
```

Comment:
This should probably be in another template with its own selector and `applyTo`: for example here, the selector is `.s_newsletter_list` while it should be `.s_newsletter_subscribe_form` :+1: